variants/README.md

```diff
@@ -69,6 +69,11 @@ The [vmware-dev](vmware-dev/Cargo.toml) variant has useful packages for local de
 It includes tools for troubleshooting as well as Docker for running containers.
 User data will be read from a mounted CD-ROM, either from a file named "user-data" or from an OVF file.
 
+### vmware-k8s-1.19: VMWare Kubernetes 1.19 node
+
+The [vmware-k8s-1.19](vmware-k8s-1.19/Cargo.toml) variant includes the packages needed to run a Kubernetes worker node as a VMWare guest.
+It supports self-hosted clusters.
+
```

Comment:
We should have the "This variant is compatible with" line, which I'm not putting as a suggestion yet because I assume we're updating to 1.20.